### PR TITLE
Jeremypw/awissu fix #137

### LINKED ISSUE #137: Parental control is not working for blocking websites and applications
I have set a list of websites and applications that I wanto to block in the accounts of my kids. However, the sites and the applications ares still accesible.

elementary OS 5.1.6 Hera

### PR DESCRIPTION
**Pull request by jeremypw**
Fixes #137 

Based on #156 by merging master and fixing lint

Pushing here as the original author has not shown any Github activity for the last two years (and little before that)

### CODE REVIEW

**Comment by jeremypw** [comment]
I have discovered that this does not work if `ufw` is enabled - presumably because `ufw` puts rules with a higher priority than this PR and accepts the url before it reaches this app's rules??

File: src/daemon/Restrictions/WebRestriction.vala [hunk 1]
```diff
@@ -35,43 +35,42 @@ namespace PC.Daemon {
...
             } catch (Error e) {
                 warning (e.message);
             }
-
-            return address_list;
+            return false;
         }
 
-        private void process_adress (string address, string option) {
+        private void process_adress (string url, string option, string filter) {
```
Comment (zeebok) [inline src/daemon/Restrictions/WebRestriction.vala:70]: Not critical to functionality but 'address' is misspelled

**Comment by zeebok** [comment]
I mean code-wise nothing seems unusual but I am also not to familiar with iptables and all that either

**Review by danirabbit — APPROVED** [review]
I'm not sure we're going to get a more comprehensive review at this time. If it seems  like it's working for some definition of, then let's do it :sweat_smile: Hopefully migrating from ufw to firewalld in the future resolves that particular issue

**Pull request merged**